[DOC] Update Management navigation interface in dashboards for UX revisions and add index management tutorial (#4550)
opensearch-project/documentation-website: _dashboards/sm-dashboards.md

  - /dashboards/admin-ui-index/sm-dashboards/
---

> # Snapshot management in Dashboards

Reviewer: ```suggestion
# Snapshot Management in Dashboards
```

> [Snapshots]({{site.url}}{{site.baseurl}}/opensearch/snapshots/index/) are backups of a cluster’s indexes and state. The state includes cluster settings, node information, index metadata (mappings, settings, templates), and shard allocation. The Snapshot Management interface in OpenSearch Dashboards provides a unified solution for taking and restoring snapshots.

Reviewer: ```suggestion
[Snapshots]({{site.url}}{{site.baseurl}}/opensearch/snapshots/index/) are backups of a cluster’s indexes and state. The state includes cluster settings, node information, index metadata (mappings, settings, templates), and shard allocation. The Snapshot Management (SM) interface in OpenSearch Dashboards provides a unified solution for taking and restoring snapshots.
```

    For example, if you’re moving from a proof of concept to a production cluster, you might take a snapshot of the former and restore it on the latter.

## Creating a repository

> Before you create a snapshot management policy, set up a repository for snapshots. 

Reviewer: ```suggestion
Before you create an SM policy, set up a repository for snapshots. 
```

    ```
1. Choose the **Add** button.

> If you need to automate snapshots creation, you can use a snapshot policy.

Reviewer: ```suggestion
If you need to automate snapshot creation, you can use a snapshot policy.
```
